MalwareAnalyzer/views/domain_check.py

```diff
@@ -91,6 +91,34 @@ def malware_check(urllist):
 # Helper Functions
 
 
+def verify_domain(checkeddom):
+    try:
+        if (len(checkeddom) > 2
+                and '.' in checkeddom
+                and (checkeddom.endswith('.') is False
+                     and re.search('[a-zA-Z0-9]', checkeddom))):
+            return(True)
+        else:
+            return(False)
+    except Exception:
+        logger.exception('[ERROR] Verifiyning Domain')
+
+
+def get_netloc(url):
+    """Get Domain."""
+    try:
+        domain = ''
+        parsed_uri = urlparse(url)
+        if not urlparse(url).scheme:
+            url = '//' + url
+            parsed_uri = urlparse(url)
+            domain = '{uri.netloc}'.format(uri=parsed_uri)
+        if verify_domain(domain):
+            return(domain)
+    except Exception:
+        logger.exception('[ERROR] Extracting Domain form URL in APK')
```

Comment:
Not just APK, but source as well so. Reword to 

`[ERROR] Extracting domain form URL`